contracts/fei/FSRX.sol

```diff
@@ -0,0 +1,96 @@
+import {ERC20} from "../external/solmate/tokens/ERC20.sol";
+import {ERC4626} from "../external/solmate/mixins/ERC4626.sol";
+import {TribeRoles} from "../core/TribeRoles.sol";
+import {CoreRef} from "../refs/CoreRef.sol";
+import {Constants} from "../Constants.sol";
+
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+library FSRXConfig {
+    address internal constant CORE_ADDRESS = address(0x8d5ED43dCa8C2F7dFB20CF7b53CC7E593635d7b9);
+    address internal constant FEI_ADDRESS = address(0x956F47F50A910163D8BF957Cf5846D573E7f87CA);
+
+    string internal constant TOKEN_NAME = "Fei Savings Rate";
+    string internal constant TOKEN_SYMBOL = "FSRX";
+
+    uint256 internal constant MAX_INTEREST_RATE = 1_000_000;
+
+    event InterestRateUpdate(uint256 newRate);
```

Comment:
Why not include the old interest rate in this event? Would make it easier for clients/frontends so track what's going on. This is unlikely to be called often, so minimal gas overhead